### DISCUSSION TITLE
Linq2db for Mongodb

### DISCUSSION THREAD

**Discussion by dogaanismail** [linq2db/linq2db#3201] [Q&A] [answered]
Does Linq2db support Mongodb implementation?

**Comment by MaceWindu** [comment, marked as answer]
Nope. List of currently supported databases could be found [here](https://github.com/linq2db/linq2db/wiki#supported-databases)
[Here](https://github.com/linq2db/linq2db/issues/1014) you can request support for new database

**Comment by dogaanismail** [comment]
Thank you for the answer.
Where can I request support for new database implementation? Is it enough to create an issue for that?

**Reply by MaceWindu** [reply]
There is a pinned issue for such requests https://github.com/linq2db/linq2db/issues/1014
